src/main/java/com/epam/ta/reportportal/ws/controller/ProjectController.java

```diff
@@ -337,6 +337,16 @@ public ProjectInfoResource getProjectInfo(@PathVariable String projectName,
 		return getProjectHandler.getAllProjectNames();
 	}
 
+	@Transactional(readOnly = true)
+	@PreAuthorize(ADMIN_ONLY)
+	@RequestMapping(value = "/names/search", method = RequestMethod.GET)
+	@ResponseBody
```

Comment:
it is @RestController so we don't need this annotation

Reply:
fixed